lib/components/util/operator-logo.tsx

```diff
@@ -3,6 +3,9 @@ import React from 'react'
 import styled from 'styled-components'
 
 const OperatorImg = styled.img`
+  &:not(:last-of-type) {
+    margin-right: 0.25ch;
```

Comment:
I think these are a little a little tight together, I wouldn't mind upping this to like .5!